src/analytics/private/sqls/redshift/event-v2.sql

```diff
@@ -51,27 +54,30 @@ CREATE TABLE IF NOT EXISTS {{schema}}.event_v2 (
     app_install_source varchar(255),
     platform varchar(255),
     project_id varchar(255) not null,
-    screen_name varchar(255),
-    screen_id varchar(255),
-    screen_unique_id varchar(255),
-    previous_screen_name varchar(255),
-    previous_screen_id varchar(255),
-    previous_screen_unique_id varchar(255),
-    page_referrer varchar(65535),
-    page_referrer_title varchar(2048),
-    page_title varchar(2048),
-    page_url varchar(65535),
-    page_url_path varchar(65535),
-    page_url_query_parameters super,
-    hostname varchar(2048),
-    latest_referrer varchar(65535),
-    latest_referrer_host varchar(2048),
-    app_start_is_first_time bool,
-    previous_view_time_msec bigint,
-    previous_view_engagement_time_msec bigint,
-    entrances bool,
-    previous_app_version varchar(255),
-    previous_os_version varchar(255),
+    screen_view_screen_name  varchar(255),
+    screen_view_screen_id  varchar(255),
+    screen_view_screen_unique_id varchar(255),
+    screen_view_previous_screen_name varchar(255),
+    screen_view_previous_screen_id  varchar(255),
+    screen_view_previous_screen_unique_id varchar(255),
+    screen_view_previous_time_msec bigint, 
+    screen_view_engagement_time_msec bigint,
+    screen_view_entrances bool,
+    page_view_page_referrer  varchar(65535),
+    page_view_page_referrer_title varchar(2048),
+    page_view_previous_time_msec bigint,
+    page_view_engagement_time_msec bigint,
+    page_view_page_title varchar(2048),
+    page_view_page_url varchar(65535),
+    page_view_page_url_path varchar(65535),
+    page_view_page_url_query_parameters super,
+    page_view_hostname varchar(2048),
+    page_view_latest_referrer varchar(65535),
+    page_view_latest_referrer_host varchar(2048),
+    page_view_is_first_time bool,
+    page_view_entrances bool,
+    upgrade_previous_app_version varchar(255),
+    upgrade_previous_os_version varchar(255),
```

Comment:
sdk_exception_message varchar(2048),
    sdk_exception_stack varchar(65535),

change to be:
    app_exception_message varchar(2048),
    app_exception_stack varchar(65535),